frontend/www/js/omegaup/components/homepage/Testimonials.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <!-- id-lint off -->
+  <div
+    class="carousel slide testimonials-carousel"
+    data-ride="carousel"
+    data-interval="8000"
+    id="testimonials"
+  >
+    <!-- id-lint off -->
+    <ol class="carousel-indicators">
+      <li
+        data-target=".testimonials-carousel"
```

Comment:
acá igual, esto funciona?

```suggestion
        data-target="#testimonials-carousel-display"
```